src/vs/workbench/contrib/terminal/browser/terminalService.ts

```diff
@@ -891,8 +893,27 @@ export class TerminalService implements ITerminalService {
 		if (!value) {
 			return;
 		}
-		await this._configurationService.updateValue(`terminal.integrated.shell.${platformKey}`, value.profile.path, ConfigurationTarget.USER);
-		await this._configurationService.updateValue(`terminal.integrated.shellArgs.${platformKey}`, value.profile.args, ConfigurationTarget.USER);
+		if ('createInstance') {
+			const launchConfig = { executable: value.profile.path, args: value.profile.args, name: value.profile.overrideName ? value.profile.profileName : undefined };
+			let instance;
+			if (keyMods?.alt) {
+				// create split
+				const activeInstance = this.getActiveInstance();
+				if (activeInstance) {
+					instance = this.splitInstance(activeInstance, launchConfig);
+				}
```

Comment:
think this won't work if the panel isn't open (no terminals) and I hold down alt. fixing